js/colourpicker.js

```diff
@@ -38,21 +38,22 @@
         show: null,
         showSpeed: 100,
 		
-		    // added by Dean Attali
+		    // added by Dean Attali modified in some cases by David Griswold
         showColour: 'both',
         allowTransparent: false,
         transparentText: 'Transparent',
         palette: 'square',
         allowedCols:
-          "#000000 #333333 #4D4D4D #666666 #7F7F7F #999999 #B3B3B3 #E5E5E5 " +
-          "#FFFFFF #27408B #000080 #0000FF #1E90FF #63B8FF #97FFFF #00FFFF " +
-          "#00868B #008B45 #458B00 #008B00 #00FF00 #7FFF00 #54FF9F #00FF7F " +
-          "#7FFFD4 #8B4500 #8B0000 #FF0000 #FF6A6A #FF7F00 #FFFF00 #FFF68F " +
-          "#F4A460 #551A8B #8B008B #8B0A50 #9400D3 #FF00FF #FF1493 #E066FF",
+          ["#333333FF", "#4D4D4DFF", "#666666FF", "#7F7F7FFF", "#999999FF", "#B3B3B3FF", "#E5E5E5FF",  
```

Comment:
WOAH and this is even cooler, looks like putting a hex code inside a code block makes github show the resulting colour beside it!